### PR TITLE
Add .circleci/config.yml

### PR DESCRIPTION
**Pull request by spacechurro**
https://trello.com/c/g4IqUkzZ/428-finish-setting-up-ci-on-forked-land-gem

### CODE REVIEW

File: land.gemspec [hunk 1]
```diff
@@ -1,56 +1,58 @@
-$:.push File.expand_path("lib", __dir__)
+$:.push File.expand_path('lib', __dir__)
```
Comment (spacechurro) [inline land.gemspec]: I would turn off this whitespace rubocop change. We don't wanna impose our stylistic guidelines on this public gem.

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -78,8 +83,8 @@
   # Filter lines from Rails gems in backtraces.
   config.filter_rails_from_backtrace!
 
-  # arbitrary gems may also be filtered via:
-  config.filter_gems_from_backtrace("rspec", "rspec-rails")
+  # # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace('rspec', 'rspec-rails')
```
Comment (spacechurro) [inline spec/rails_helper.rb]: How come this config got shut off?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -103,53 +112,51 @@
...
-  #     --seed 1234
-  config.order = :random
-
-  # Seed global randomization in this process using the `--seed` CLI option.
-  # Setting this allows you to use `--seed` to deterministically reproduce
-  # test failures related to randomization by passing the same `--seed` value
-  # as the one that triggered the failure.
-  Kernel.srand config.seed
-=end
+  # The settings below are suggested to provide a good initial experience
```
Comment (spacechurro) [inline spec/spec_helper.rb]: I would turn off whatever rubocop rule enforced this if we can. Its kinda funky that this is double commented.

**Review by spacechurro — COMMENTED** [review]
I apparently can't "request changes" on my own github request. Heh. 

This is sweet! A few minor tweaks and we should be golden. Thx!

**Comment by spacechurro** [comment]
@andreaMT15 I added some notes here, but didnt tag you on it so you might not have received an update. Not urgent, but shoot me a note when I can take another look at this PR.

**Comment by andreaMT15** [comment]
@spacechurro You can take a look at the PR again.

File: .gitignore [hunk 4]
```diff
@@ -1,6 +1,6 @@
-Gemfile.lock
```
Comment (spacechurro) [inline .gitignore]: Mah bad. Missed this. We're not supposed to check in Gemfile.lock for gems.

https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

File: spec/rails_helper.rb [hunk 5]
```diff
@@ -1,25 +1,30 @@
 # frozen_string_literal: true
 
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV['RAILS_ENV'] ||= "test"
+ENV['RAILS_ENV'] ||= 'test'
```
Comment (spacechurro) [inline spec/rails_helper.rb]: Still seeing double quote to single quote changes. Can we revert these and turn off this cop?

File: spec/spec_helper.rb [hunk 6]
```diff
@@ -112,19 +121,16 @@
   # aliases for `it`, `describe`, and `context` that include `:focus`
   # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
   config.filter_run_when_matching :focus
-
```
Comment (spacechurro) [inline spec/spec_helper.rb]: Can we undo these white space changes? Looking to have as small of a diff as possible in case we wanna merge back.

**Review by spacechurro — COMMENTED** [review]
@andreaMT15 Got a few more changes. Can you take a look?

**Comment by andreaMT15** [comment]
@spacechurro I think I got all the changes now. Can you take a look again?

File: Gemfile.lock [hunk 7]
```diff
@@ -0,0 +1,135 @@
+PATH
```
Comment (spacechurro) [inline Gemfile.lock]: We still gotta delete this file since we shouldn't check in the lock file for gems.

**Review by spacechurro — COMMENTED** [review]
@andreaMT15 We're close. 1 last fix and we should be good to go.

**Comment by andreaMT15** [comment]
@spacechurro made the last fix. 😃    
Can you take another look?

**Review by spacechurro — COMMENTED** [review]
👍

**Pull request merged**